RileyLinkKit/PumpOps.swift

```diff
@@ -55,6 +55,23 @@ public class PumpOps {
         }
     }
     
+    public func readSettings(_ completion: @escaping (Either<ReadSettingsCarelinkMessageBody, Error>) -> Void)  {
+        device.runSession(withName: "Read pump settings") { (session) -> Void in
+            let ops = PumpOpsSynchronous(pumpState: self.pumpState, session: session)
+            do {
+                let response: ReadSettingsCarelinkMessageBody = try ops.messageBody(to: .readSettings)
+                DispatchQueue.main.async { () -> Void in
```

Comment:
Are you referring to the dispatch on main thread? I thought I'd fit the pattern and keep consistent behavior. 

Or are you asking if I actually need to expose readSettings?  I thought it might be useful if we want to diagnose temp basal failures, and for setting Loop configuration from pump.